devReportConfig.txt

```diff
@@ -1,4 +1,4 @@
-;[
```

Comment:
This was added by prettifier in a previous PR, that's also the reason I changed the extension

Reply:
Would it work if the extension was `.json`?

Reply:
It's not a valid json because it's lacking quotes on the keys